packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -196,21 +236,26 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
     // When the dropdown opens
     if (isOpen) {
       inputRef.current?.focus(); // Focus the input
-      setFilteredOptions(optionsRef.current); // Reset the filtered list
-      dispatch(['focus']);
+      // Reset the filtered list (does not reset when multiple enabled)
+      !multiple && setFilteredOptions(optionsRef.current);
```

Comment:
Just want to confirm that `isSingleOnChange` or `!isMultipleEnabled` is not needed here

Reply:
setFilteredOptions does not use values that changes types with multiple enabled. Therefore, isMultipleEnabled is not required.

Reply:
Edit: changed code based on comment on line 243.